esy.json

```diff
@@ -12,9 +12,8 @@
   },
   "esy": {
     "build": "dune build -p #{self.name}",
-    "buildEnv": { "ODOC_SYNTAX": "re" },
     "buildsInSource": "_build",
-    "release": { "bin": ["Bin", "UsePpx"] }
+    "release": { "bin": ["tailwind-ppx", "use-tailwind-ppx"] }
```

Comment:
I get this warning when running `esy npm-release`:

```ocaml
File "/var/folders/y_/flbz1jx93jb37tz90qfm_qxds9tdgw/T/esy-f79275/tailwind-ppx.ml", line 1:
Warning 24: bad source file name: "Tailwind-ppx" is not a valid module name.
File "/var/folders/y_/flbz1jx93jb37tz90qfm_qxds9tdgw/T/esy-f79275/use-tailwind-ppx.ml", line 1:
Warning 24: bad source file name: "Use-tailwind-ppx" is not a valid module name.
```

I'm planning to open a separate PR to rename them (should fix #94)

Reply:
Hmm I'm a bit worried about these warnings. Is there any way we can get rid of them? Using hyphens in module names isn't allowed, so I'm worried things might break if we don't fix it. I'm thinking it has something to do with the line you highlighted:

```
"release": { "bin": ["tailwind-ppx", "use-tailwind-ppx"] }
```

On another note, doesn't this line say where to find the modules we're releasing? The modules we want are `bin/Bin` and `bin/UsePpx` so I'm struggling to see how this will work -- perhaps you could explain this change? Thanks so much.

Reply:
Yeah, I have been confused as well. After some trial and error, it turns out that it should match the `public_name` property from the `dune` file.

As for the warnings. I agree that we should get rid of them. As I said, I'm planning to address that in another PR (once this one is merged).

Reply:
Yeah that'd be nice -- if everything works fine though we can live with these warnings.